worker-core/src/main/java/com/hpe/caf/worker/core/StreamingWorkerWrapper.java

```diff
@@ -53,11 +55,13 @@ public StreamingWorkerWrapper(final WorkerTaskImpl workerTask)
     @Override
     public void run()
     {
+        final String workerFriendlyName = !Strings.isNullOrEmpty(System.getenv(CAF_WORKER_NAME)) ?
```

Comment:
Let's move the retrieval of the env variable to a static final.
run() gets called for each message meaning this bit of logic could be repeated 100s of times per second.